### PR TITLE
Improve GraphHopperRouting

### PR DESCRIPTION
**Pull request by boldtrn**
I added some improvements to the GraphHopperRouting implementation. I made it possible to access /info and /i18n, also I added coordinates to the instructions to make it easier for users to use them. Therefore, this PR solves #20.

BTW: I increased the timeout for the MapMatching since I ran into a couple of timeouts.

### CODE REVIEW

**Comment by boldtrn** [comment]
It would be good if we could reference the whole leg for the instructions, instead of just the first point.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "graphhopper-js-api-client",
-  "version": "0.9.0-1",
+  "version": "0.9.0-2",
```
Comment (karussell) [inline package.json]: Can we use 'dot' notation for everything like 0.9.0.2?

**Comment by karussell** [comment]
Looks good. Do you want a new PR for referencing the whole geometry? (I.e. should I wait or merge :) ?)

File: package.json (hunk 1 above)
Comment (boldtrn) [inline package.json]: Unfortunately not. This is not the semantic version standard and is not accepted by npm.

Comment (karussell) [inline package.json]: Oh, yes. They are probably right, e.g. Linux also uses X.Y.Z-A

**Comment by boldtrn** [comment]
> Do you want a new PR for referencing the whole geometry? (I.e. should I wait or merge :) ?)

Just added it :)

File: package.json (hunk 1 above)
Comment (boldtrn) [inline package.json]: http://semver.org/ :)

**Pull request merged**